source/custom/nixnet_service.custom.cpp

```diff
@@ -803,4 +805,68 @@ ::grpc::Status NiXnetService::DbGetDatabaseList(::grpc::ServerContext* context,
   }
 }
 
-}  // namespace nixnet_grpc
\ No newline at end of file
+// template <>
```

Comment:
@astarche I couldn't find a way to make this template instantiation. Since this method is specific to XNET, I guess this should be fine right?